### DISCUSSION TITLE
Login to GitHub

### DISCUSSION THREAD

**Discussion by saharsalman** [community/community#37967] [Other Feature Feedback, Questions, & Ideas]
Asalam u Alaikum, 

Basically I had an account linked with my organization email. Now I left the office and my email account will be disabled. GitHub is sending the code to that email but I don't have an access of that account.

I have my personal development projects on the GitHub. 
Do I have any option to access my old account?

**Comment by airtower-luna** [comment]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!
